app/src/main/java/com/geeksville/mesh/util/LocationUtils.kt

```diff
@@ -181,7 +183,7 @@ fun latLongToMeter(
     lat_b: Double,
     lng_b: Double
 ): Double {
-    val pk = (180 / 3.14169)
+    val pk = (180 / PI)
```

Comment:
Kotlin has a built-in `Math.PI` constant you can use. let me know what you prefer.

Reply:
Cool, Ill use the built in one instead.